jenkins_jobs/trigger_host_os_build_from_builds_repo.groovy

```diff
@@ -38,7 +38,7 @@ job('trigger_host_os_build_from_builds_repo') {
 	}
 	parameters {
 	  predefinedProps([BUILDS_REPO_REFERENCE: '$sha1',
-			   VERSIONS_REPO_REFERENCE: 'master'])
+			   VERSIONS_REPO_REFERENCE: 'origin/master'])
```

Comment:
There are other jobs mentioning only "master". Isn't it better to modify them too?
e.g. jenkins_jobs/trigger_yamllint_from_versions_repo.groovy, jenkins_jobs/trigger_weekly_host_os_build.groovy: